rust/private/rust.bzl

```diff
@@ -110,12 +111,13 @@ def _rust_library_impl(ctx):
         output_hash,
     )
     rust_lib = ctx.actions.declare_file(rust_lib_name)
+    crate_name = ctx.label.name.replace("-", "_")
```

Comment:
can remove one more by passing crate_name to L108

Reply:
One of them is `ctx.attr.name`, the other is `ctx.label.name`. Are those interchangeable?

Reply:
I'm 99% sure they are, but not entirely.. I think the difference is that ctx.label is the full label